<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameMaster">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frameImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>351</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="txtBoxSens">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.225</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxFrames">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxHeight">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>600</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Frames:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Iterations:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxIters">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>5000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sens:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxWidth">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>800</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameParameter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="txtBoxDD">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1.2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>40</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxBB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1.9</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>B:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>C:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>70</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxA">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>-1.7</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxD">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>-0.8</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxAA">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>-0.6</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1.2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>102</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.9</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>12</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxCC">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>1.8</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameColor">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>221</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>From:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxGreen">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>255</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxBlue">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxBlue2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxRed2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>255</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ttxBoxRed">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBoxGreen2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>125</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>HUE shift</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2color grad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>From file</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Coloring mode:</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Useless big button</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRender">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Render</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonClose">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Close!</string>
      </property>
     </widget>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>531</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>150</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>Quit</string>
    </property>
    <addaction name="actionYep_quit_now"/>
   </widget>
   <addaction name="menuQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionYep_quit_now">
   <property name="text">
    <string>Yep, quit now!</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>txtBoxA</tabstop>
  <tabstop>txtBoxAA</tabstop>
  <tabstop>txtBoxB</tabstop>
  <tabstop>txtBoxBB</tabstop>
  <tabstop>txtBoxC</tabstop>
  <tabstop>txtBoxCC</tabstop>
  <tabstop>txtBoxD</tabstop>
  <tabstop>txtBoxDD</tabstop>
  <tabstop>ttxBoxRed</tabstop>
  <tabstop>txtBoxGreen</tabstop>
  <tabstop>txtBoxBlue</tabstop>
  <tabstop>txtBoxRed2</tabstop>
  <tabstop>txtBoxGreen2</tabstop>
  <tabstop>txtBoxBlue2</tabstop>
  <tabstop>txtBoxWidth</tabstop>
  <tabstop>txtBoxHeight</tabstop>
  <tabstop>txtBoxFrames</tabstop>
  <tabstop>txtBoxIters</tabstop>
  <tabstop>txtBoxSens</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>buttonRender</tabstop>
  <tabstop>buttonClose</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
